packages/acs-ui-common/src/conditional-compilation-sample/index.tsx

```diff
@@ -123,6 +123,14 @@ export function d(e: number, f?: number, g?: number): void {
   console.log(f, g);
 }
 
```

Comment:
Would be nice to add a trailing comment here like
  console.log(f, g);
} // this is a trailing comment which is above conditional compile comment and wouldn't be removed